integrationTests/docker/Dockerfile_java11

```diff
@@ -8,17 +8,20 @@ ENV SDK_TOOLS=sdk-tools-linux-4333796.zip
 
 #Installing android sdk
 RUN echo "Installing android-sdk" \
-  && cd /opt && mkdir android-sdk-linux && cd android-sdk-linux && mkdir add-ons platforms platform-tools temp \
-  && cd /opt/android-sdk-linux && wget https://dl.google.com/android/repository/$SDK_TOOLS && unzip $SDK_TOOLS && rm $SDK_TOOLS
+  && mkdir --parent /opt/android-sdk-linux/{add-ons,platforms,platform-tools,temp} \
+  && cd /opt/android-sdk-linux \
+  && curl --remote-name https://dl.google.com/android/repository/$SDK_TOOLS \
+  && unzip $SDK_TOOLS \
+  && rm $SDK_TOOLS
 
 RUN mkdir /usr/share/java
 
-RUN wget https://repo1.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.1/jaxb-api-2.3.1.jar -O /usr/share/java/jaxb-api.jar \
-&& wget https://repo1.maven.org/maven2/javax/xml/jaxb-impl/2.1/jaxb-impl-2.1.jar -O /usr/share/java/jaxb-impl.jar \
-&& wget https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-core/2.3.0.1/jaxb-core-2.3.0.1.jar -O /usr/share/java/jaxb-core.jar \
-&& wget https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-jxc/2.3.2/jaxb-jxc-2.3.2.jar -O /usr/share/java/jaxb-jxc.jar \
-&& wget https://repo1.maven.org/maven2/org/glassfish/jaxb/jaxb-xjc/2.3.2/jaxb-xjc-2.3.2.jar -O /usr/share/java/jaxb-xjc.jar \
-&& wget https://repo1.maven.org/maven2/javax/activation/activation/1.1.1/activation-1.1.1.jar -O /usr/share/java/activation.jar
+RUN curl https://repo1.maven.org/maven2/javax/xml/bind/jaxb-api/2.3.1/jaxb-api-2.3.1.jar --output /usr/share/java/jaxb-api.jar \
```

Comment:
Shouldn't the dependencies be loaded through dependency management of Gradle or Maven?

Reply:
These libraries are used by the android framework.
I guess, at the time of writing this Dockerfile android did not support java 11.
Nowadays it changes. It could be the next step to use the latest android sdk for the jdk 11 environment.